.pfnci/windows/_flexci.ps1

```diff
@@ -0,0 +1,43 @@
+function ActivatePython($version) {
+    if ($version -eq "3.5") {
+        $pydir = "Python35"
+    } elseif ($version -eq "3.6") {
+        $pydir = "Python36"
+    } elseif ($version -eq "3.7") {
+        $pydir = "Python37"
+    } elseif ($version -eq "3.8") {
+        $pydir = "Python38"
+    } elseif ($version -eq "3.9") {
+        $pydir = "Python39"
+    } else {
+        throw "Unsupported Python version: $version"
+    }
+    $Env:PATH = "C:\Development\Python\$pydir;C:\Development\Python\$pydir\Scripts;" + $Env:PATH
+}
+
+function ActivateCUDA($version) {
```

Comment:
I guess this is just for dealing with all the configs in the windows image and will ease a lot of the backport effort.
So I don't mind having unsupported versions in here.